evernode-governor-hook/src/governor.c

```diff
@@ -851,6 +830,44 @@ int64_t hook(uint32_t reserved)
                 PERMIT_M("Evernode: Piloted mode candidate status changed.", vote_status);
             }
         }
+
+        if (redirect_op_type == OP_CHANGE_CONFIGURATION)
+        {
+            SET_UINT_STATE_VALUE(DEF_MOMENT_SIZE, CONF_MOMENT_SIZE, "Evernode: Could not initialize state for moment size.");
+            SET_UINT_STATE_VALUE(DEF_MINT_LIMIT, CONF_MINT_LIMIT, "Evernode: Could not initialize state for mint limit.");
+            SET_UINT_STATE_VALUE(DEF_FIXED_REG_FEE, CONF_FIXED_REG_FEE, "Evernode: Could not initialize state for fixed reg fee.");
+
+             // Take the moment size from config.
+            uint16_t moment_size = 0;
+            GET_CONF_VALUE(moment_size, CONF_MOMENT_SIZE, "Evernode: Could not get moment size.");
+            TRACEVAR(moment_size);
```

Comment:
![image](https://user-images.githubusercontent.com/26274844/233322761-144fd2a7-9772-40a4-b22a-37806b96b7eb.png)

this is outputting the value correctly